scaleway/resource_vpc_gateway_network.go

```diff
@@ -124,18 +127,29 @@ func resourceScalewayVPCGatewayNetworkCreate(ctx context.Context, d *schema.Reso
 		Timeout:       scw.TimeDurationPtr(gatewayWaitForTimeout),
 		RetryInterval: &retryInterval,
 	}, scw.WithContext(ctx))
-
-	if err != nil && !is404Error(err) {
+	if err != nil {
```

Comment:
Why do we ignore this case now?

Reply:
We need explicitly that the gateway exist.